Controller/CRUDController.php

```diff
@@ -234,7 +234,8 @@ public function editAction($id = null)
         // the key used to lookup the template
         $templateKey = 'edit';
 
-        $id = $id ?: $request->get($this->admin->getIdParameter());
+        // If the $id variable is provided AND the child ID is not needed, do not reload it.
```

Comment:
> which child ?

Any child.

Ref: https://sonata-project.org/bundles/admin/3-x/doc/reference/architecture.html#create-child-admins

> how do you detect it is not needed, exactly ?

By the result of `$this->admin->getIdParameter()`:

* `id` -> Concerns the main admin, same as the given one as parameter
* `childId` -> Concerns a child. Have to reload because the given `$id` is always the id of the main admin subject.

See line below. :wink: